### ISSUE TITLE
Evaluate Runtime Parameters Centrifuge

### ISSUE THREAD

**Issue by mustermeiszer** [centrifuge/centrifuge-chain#563]
Need to evaluate the Centrifuge runtime parameters of 
* Council
* Treasury
* ...

This needs to happen before we enable calls for these.

**Comment by mustermeiszer** [comment]
@mikiquantum I think this is already done as the chain is live. Will close therefore.
